pkg/v19/key/key.go

```diff
@@ -92,6 +110,7 @@ func CommonChartSpecs() []ChartSpec {
 			ChannelName:     "0-4-stable",
 			ChartName:       "kubernetes-kube-state-metrics-chart",
 			ConfigMapName:   "kube-state-metrics-values",
+			HasAppCR:        true,
```

Comment:
There won't be a migration path for apps on clusterapi controllers. But the key functions are shared and we need this for the legacy controllers migration path.